backend/handler/password.go

```diff
@@ -49,6 +56,14 @@ func (h *PasswordHandler) Set(c echo.Context) error {
 		return dto.NewHTTPError(http.StatusBadRequest, "failed to parse userId as uuid").SetInternal(err)
 	}
 
+	pwBytes := []byte(body.Password)
+	if utf8.RuneCountInString(body.Password) < h.cfg.MinPasswordLength { // use utf8.RuneCountInString, so utf8 characters would count as 1
```

Comment:
Why check for RuneCount? I thought the bcrypt algorithm operates on bytes, so if you use utf-8 characters you could potentially break the password length limit.

Reply:
Yes, thats true, but if you want to show the user, whats wrong, you don't want to bother the user with "bytes", because most of the user don't know what the term "bytes" means and don't understand what they should do to avoid that error. So I think it's better to check for characters and not for bytes here.